docs/experiments_and_process_variables.md

```diff
@@ -28,11 +28,13 @@
 
 ### ProcessVariable
 
-| Column name         | type                              | primary key | unique | nullable | default value |
-|---------------------|-----------------------------------|-------------|--------|----------|---------------|
-| pv_string           | String(100)                       | True        | True   | False    | ---           |
-| human_readable_name | String(100)                       | False       | False  | True     | ---           |
-| experiment_short_id | ForeignKey("experiment.short_id") | False       | False  | False    | ---           |
+| Column name           | type                                 | primary key | unique | nullable | default value |
+|-----------------------|--------------------------------------|-------------|--------|----------|---------------|
+| pv_string             | String(100)                          | True        | True   | False    | ---           |
+| human_readable_name   | String(100)                          | False       | False  | True     | ---           |
+| experiment_short_id   | ForeignKey("experiment.short_id")    | False       | False  | False    | ---           |
+| default_threshold_min | db.Column(db.Integer, nullable=True) | False       | False  | True     | ---           |
```

Comment:
Please remove ", nullable=True" for default_threshold_min and default_threshold_max

Reply:
done